core/templates/dev/head/domain/topic/topic-update.service.ts

```diff
@@ -70,7 +72,7 @@ angular.module('oppia').factory('TopicUpdateService', [
       _applyChange(topic, CMD_UPDATE_TOPIC_PROPERTY, {
         property_name: propertyName,
         new_value: angular.copy(newValue),
-        old_value: angular.copy(oldValue)
+        old_value: angular.copy(oldValue) || null
```

Comment:
Why this change? `oldValue` should never be  undefined, right?

Reply:
This is to support topics that were created before these changes go in. Since older topics may not have oldValue (ie. it will be undefined) for abbreviated name and thumbnail fields, the changelist JSON sent to the backend might not contain old_value key and would throw a backend error.